artifacts_test.py

```diff
@@ -227,6 +230,23 @@ def verify_docker_locale_settings(self) -> None:
             configured_value = locale_settings.get(setting_key)
             self.assertEqual(configured_value, expected_value)
 
+    def verify_docker_latest_match_release(self) -> None:
+        for product in ['scylla', 'scylla-enterprise']:
+            product: ScyllaProduct
+            ami_latest_version = get_latest_scylla_release(product=product)
+
+            url = 'https://hub.docker.com/v2/repositories/scylladb/{}/tags/{}'
+            docker_latest = requests.get(url.format(product, 'latest')).json()
+            docker_release = requests.get(url.format(product, ami_latest_version)).json()
+            self.log.debug('latest info: %s', pprint.pformat(docker_latest))
+            self.log.debug('%s info: %s ', ami_latest_version, pprint.pformat(docker_release))
+
+            latest_digests = set(image['digest'] for image in docker_latest['images'])
+            release_digests = set(image['digest'] for image in docker_release['images'])
+
+            assert latest_digests == release_digests, \
+                f"latest != {ami_latest_version}, images digest differs [{latest_digests}] != [release_digests]"
```

Comment:
release_digests are not substituted. Is it like intended?

Reply:
you are right, need to fix it

Reply:
fixed